libs/commons-database/src/main/kotlin/gov/cdc/ocio/database/mongo/MongoRepository.kt

```diff
@@ -0,0 +1,76 @@
+package gov.cdc.ocio.database.mongo
+
+import gov.cdc.ocio.database.persistence.Collection
+import com.mongodb.*
+import com.mongodb.client.MongoClient
+import com.mongodb.client.MongoClients
+import com.mongodb.client.MongoDatabase
+import gov.cdc.ocio.database.persistence.ProcessingStatusRepository
+import org.bson.BsonDocument
+import org.bson.BsonInt64
+
+
+/**
+ * Mongo repository implementation
+ *
```

Comment:
`@param` tags for uri and databaseName are missing. `@return` tag is also missing.

Reply:
Fixed